src/backend/src/services/teams.services.ts

```diff
@@ -152,8 +156,20 @@ export default class TeamsService {
 
     if (team.dateArchived) throw new HttpException(400, 'Cannot edit the head of an archived team');
 
-    if (newHead && team.members.map((user) => user.userId).includes(newHead?.userId))
-      throw new HttpException(400, 'Error: Team head cannot be a member');
+    // if the new head is a current member on the team, this will remove them as a member
+    if (newHead && team.members.map((user) => user.userId).includes(userId)) {
+      const newTeamArr: number[] = team.members
+        .filter((member) => member.userId !== newHead.userId)
```

Comment:
just use userId here instead of newHead.userId, since that's what you're doing in the if statement